vhdbuilder/packer/install-dependencies.sh

```diff
@@ -218,6 +218,7 @@ EOF
 installBcc
 cat << EOF >> ${VHD_LOGS_FILEPATH}
   - bcc-tools
+  - bcc-examples
```

Comment:
No need to add `- bcc-examples` to this list, especially since this logging statement is for both Ubuntu and Mariner. 
We already print out the package list into the log at the end of the AgentBaker run.
```suggestion
```